avocado/core/test.py

```diff
@@ -558,9 +559,10 @@ def set_runner_queue(self, runner_queue):
         """
         Override the runner_queue
         """
-        self.assertTrue(self.__runner_queue is None, "Overriding of runner_"
-                        "queue multiple times is not allowed -> old=%s new=%s"
-                        % (self.__runner_queue, runner_queue))
+        if self.__runner_queue is not None:
+            raise ValueError("Overriding of runner_queue multiple "
+                             "times is not allowed -> old=%s new=%s"
+                             % (self.__runner_queue, runner_queue))
```

Comment:
The same here, it's not about the value, it's about the procedure. RuntimeError woud be IMO more appropriate.

Reply:
ACK.